src/btree/bt_split.c

```diff
@@ -2184,3 +2187,25 @@ __wt_split_rewrite(WT_SESSION_IMPL *session, WT_REF *ref, WT_MULTI *multi)
 err:	__split_multi_inmem_fail(session, page, new);
 	return (ret);
 }
+
+/*
+ * __page_split_timing_stress --
+ *  Optionally add a 1000 micro seconds delay to simulate the race
+ *  conditions in page split for debug purposes. The purpose of
+ *  this is to uncover the race conditions in page split.
+ */
+static void
+__page_split_timing_stress(WT_SESSION_IMPL *session)
+{
+       WT_CONNECTION_IMPL *conn;
```

Comment:
alignment in not correct here. There are 7 spaces followed by the text.
We use tabs to align different levels. and 4 spaces in addition to the tabs if the line wraps over.
Works best if tabs are set to 8 spaces in the text editor.